kategory-effects/src/main/kotlin/kategory/effects/data/IO.kt

```diff
@@ -114,7 +68,180 @@ sealed class IO<out A> : IOKind<A> {
         inline fun <reified A> semigroup(SG: Semigroup<A> = kategory.semigroup()): IOSemigroupInstance<A> = IOSemigroupInstanceImplicits.instance(SG)
 
         inline fun <reified A> monoid(SM: Monoid<A> = kategory.monoid()): IOMonoidInstance<A> = IOMonoidInstanceImplicits.instance(SM)
+
+        fun <A, B> merge(
+                op1: () -> A,
+                op2: () -> B): IO<Tuple2<A, B>> = applicative().tupled(
+                invoke(op1),
+                invoke(op2)).ev()
+
+        fun <A, B, C> merge(
+                op1: () -> A,
+                op2: () -> B,
+                op3: () -> C): IO<Tuple3<A, B, C>> = applicative().tupled(
+                invoke(op1),
+                invoke(op2),
+                invoke(op3)).ev()
+
+        fun <A, B, C, D> merge(
+                op1: () -> A,
+                op2: () -> B,
+                op3: () -> C,
+                op4: () -> D): IO<Tuple4<A, B, C, D>> = applicative().tupled(
+                invoke(op1),
+                invoke(op2),
+                invoke(op3),
+                invoke(op4)).ev()
+
+        fun <A, B, C, D, E> merge(
```

Comment:
Regarding codestyle we should follow the same conventions as in https://github.com/kategory/kategory/blob/master/kategory-core/src/main/kotlin/kategory/typeclasses/Applicative.kt. Alternatively we can change the ones in Applicative to match these one. I'll leave up to you.

Reply:
I don't see one clear way of formatting it in`Applicative.kt`.